css/reset.css

```diff
@@ -18,7 +18,7 @@ menu, nav, output, ruby, section, summary,
 time, mark, audio, video {
 	margin: 0;
 	padding: 0;
-	border: 0;
+	/* border: 0; */
```

Comment:
In general, don't modify `reset.css`; instead, create your own style definitions in `style.css` that will get to win because you import that file second.